src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothAdapter.cs

```diff
@@ -401,7 +401,7 @@ public static void Disable()
         }
 
         /// <summary>
-        /// Enables the discoverable mode.
+        /// Enables the discoverable mode for the duration.
```

Comment:
This API is no duration parameter.

Reply:
myroot is correct. You have to change the summary on line 418.